Content.Shared/Standing/StandingStateComponent.cs

```diff
@@ -20,5 +25,20 @@ public sealed partial class StandingStateComponent : Component
         /// </summary>
         [DataField, AutoNetworkedField]
         public List<string> ChangedFixtures = new();
+
+        [DataField("lie-down-action")]
```

Comment:
Нарушение правил нейминга, к тому же указывание названий переменных для сериализатора таким образом устарело, и в прототипах можно писать lieDownAction: 
```suggestion
        [DataField]
```